README.md

```diff
@@ -16,42 +16,37 @@ A bunch of developers from Norway that wanted to help out. This is not an offici
 ## Mentions in the media
 - https://www.tu.no/artikler/utviklere-om-korona-register-vi-kastet-oss-rundt-og-laget-dette/487568
 
-## Hvordan kan jeg bidra?
+## How can I contribute?
+Join our Telegram group chat here: https://t.me/onzecorona or reach out on kontakt@bustbyte.no
 
-### Slack
-Vi har en felles Slack-kanal du kan bli medlem av. Her prøver vi å koordinere utviklingen så godt vi kan via. en VoIP-samtale.
-
-Send e-postadressen din til kontakt@bustbyte.no, så legger vi deg til snarest. 
-
-### GitHub Issues
 Klikk på "Issues" i menyen over for å se hva du kan bidra med.
```

Comment:
Still in Norwegian.

Reply:
sorry, didn't notice your comments. I'll open a new PR.